src/containers/dashboard.js

```diff
@@ -0,0 +1,134 @@
+/*
+This container is responsible for displaying the available details of user and the product details.
+@axios - this is used for fetching data from the JSON file
+*/
+import React from "react";
+import axios from "axios";
+import Button from "../components/button";
+import Accordion from "../components/accordion";
+import Header from "../components/header";
+import { connect } from "react-redux";
+
+class Dashboard extends React.Component {
+  constructor() {
+    super();
+    this.goToEditProfilePage = this.goToEditProfilePage.bind(this);
+  }
+  goToEditProfilePage() {
+    //navigation to edit profile page
+    this.props.history.push("/editProfile");
+  }
+  componentWillMount() {
```

Comment:
I think react will make this lifecycle method obselete, try making use of componentDidMount instead